templates/company-board/README.md

```diff
@@ -2,12 +2,20 @@
 
 ## Usage
 
+Create new tokens for the company-board entity:
+
+```
+template.newTokens()
+```
+
+Create a new company with board entity:
+
 ```
-company.prepareInstance(id)
-company.setupInstance(boardHolders, shareHolders, shareStakes)
+template.prepareInstance(name)
+template.setupInstance(boardHolders, shareHolders, shareStakes)
```

Comment:
I think this hasn't been answered:

https://github.com/aragon/dao-templates/pull/98/files#r305383821

Reply:
Answered in the thread :)